.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: node_js
+node_js:
+  - 0.10
+cache:
+  directories:
+  - node_modules
+before_script:
+  - npm install -g grunt-cli
+  - npm install -g mocha
+after_script:
+  - istanbul cover _mocha --report lcovonly && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
We added coveralls to give us and other work on this visibility on coverage beyond the terminal. Also be able to link to it in PR's like the comment above from coveralls.

I might be wrong but I thought istanbul was command line only and has no links or hooks into GitHub or a web dashboard?